assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,13 +58,35 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const FlexGrowRangeInput = styled(RangeInput)`
+  flex-grow: 1;
+`;
+
+const Rectangle = styled.button.attrs(({ disabled }) => ({
+  disabled,
+}))`
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: unset;
+  background: transparent;
+  height: auto;
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 32px;
+  cursor: pointer;
   color: ${({ theme }) => theme.colors.fg.v1};
-  ${({ isLarge }) => (isLarge ? 'margin-left' : 'margin-right')}: 20px;
+
+  &:active {
+    outline: none;
+  }
+
+  ${({ disabled }) =>
+    disabled &&
+    `
+		pointer-events: none;
```

Comment:
This is unnecessary. Disabled buttons are already disabled.

Reply:
Correct. I changed it to disabled status:
`disabled: {...}`